### PR TITLE
Logs should not be hidden behind a logger.info

### LINKED ISSUE #5901: How can I check the loss during pretraing huggingface/transformers
thanks in advance.
I trained roberta model from scratch.
But I cant  check the training loss during pretraining.
I did it by refering below link.
https://colab.research.google.com/github/huggingface/blog/blob/master/notebooks/01_how_to_train.ipynb
in above link, loss is printed every 500 steps,
but when I did, there is no loss print.

Iteration: 100%|█████████▉| 20703/20711 [4:42:54<00:07,  1.14it/s][A

Iteration: 100%|█████████▉| 20704/20711 [4:42:54<00:05,  1.24it/s][A

Iteration: 100%|█████████▉| 20705/20711 [4:42:55<00:05,  1.20it/s][A

Iteration: 100%|█████████▉| 20706/20711 [4:42:56<00:04,  1.18it/s][A

Iteration: 100%|█████████▉| 20707/20711 [4:42:57<00:03,  1.19it/s][A

Iteration: 100%|█████████▉| 20708/20711 [4:42:58<00:02,  1.16it/s][A

Iteration: 100%|█████████▉| 20709/20711 [4:42:59<00:01,  1.14it/s][A

Iteration: 100%|█████████▉| 20710/20711 [4:43:00<00:00,  1.13it/s][A

Iteration: 100%|██████████| 20711/20711 [4:43:00<00:00,  1.45it/s][A
Iteration: 100%|██████████| 20711/20711 [4:43:00<00:00,  1.22it/s]

Epoch: 100%|██████████| 13/13 [61:14:16<00:00, 16952.06s/it]  
Epoch: 100%|██████████| 13/13 [61:14:16<00:00, 16958.16s/it]
compress roberta.20200717.zip on ./pretrained
save roberta.20200717.zip on minio(petcharts)

stackoverflow link
https://stackoverflow.com/questions/62988081/checking-pretraining-loss-in-huggingface-transformers

### PR DESCRIPTION
**Pull request by LysandreJik**
Currently the logs that are printed when the global step is a multiple of the `logging_steps` are printed using a `logger.info`. In the case where a user did not set its logging level to INFO, these logs are not shown, even if the user has set a `logging_steps > 0`. This PR fixes that by putting back a print statement.

Closes https://github.com/huggingface/transformers/issues/5901

### CODE REVIEW

**Review by sgugger — APPROVED** [review]
Thanks!
